### PR TITLE
Refresh README.md

### PR DESCRIPTION
**Pull request by daknob**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,36 +124,52 @@ The list of all Hack Sessions held by date:
...
 ## Org Meetups
 
-An org meetup is a simple get-together, **open to every member of our community** who would like to contribute / discuss some organizational issues, future topics etc. Also a good opportunity to chat about all-things-tech :)
+An org meetup is a simple get-together, **open to every member of our
+community**, who would like to contribute / discuss some organizational issues,
+future topics etc. Also a good opportunity to chat about all-things-tech :)
 
-DevStaff is an open community. Open, as in Open Source. If you don't like anything about the community you can change it! 
+DevStaff is an open community. Open, as in Open Source. If you don't like
+anything about the community you can change it!
```
Comment (kouts) [inline README.md:152]: maybe we can rephrase this to "If you don't like `something` about the community you can change it"

**Review by kouts — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (daknob) [reply README.md]: Done.

**Review by kouts — APPROVED** [review]
